docs/UserGuide.md

```diff
@@ -130,15 +130,24 @@ If your changes to the data file makes its format invalid, AddressBook will disc
 
 Adds a person to the address book.
 
-Format: `add n/NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS [t/TAG]…​`
+Format: `add n/NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS [t/TAG]…​ b/BIRTHDAY i/INCOME ra/RISK_APPETITE`
 
 <div markdown="span" class="alert alert-primary">:bulb: **Tip:**
 A person can have any number of tags (including 0)
 </div>
 
+<div markdown="span" class="alert alert-primary">:bulb: **Tip:**
+The format for entering a birthday is "YYYY-MM-DD". The dates entered must be valid.
+</div>
+
+<div markdown="span" class="alert alert-primary">:bulb: **Tip:**
+The `Risk Appetite` can be classified as High, Medium or Low. They are represented by characters "H", "M" and
+"L" respectively. e.g: "ra/H". All other values will be ignored.
+</div>
+
 Examples:
-* `add n/John Doe p/98765432 e/johnd@example.com a/John street, block 123, #01-01`
-* `add n/Betsy Crowe t/friend e/betsycrowe@example.com a/Newgate Prison p/1234567 t/criminal`
+* `add n/John Doe p/98765432 e/johnd@example.com a/John street, block 123, #01-01, b/2000-05-05, i/50000, ra/L`
+* `add n/Betsy Crowe t/friend e/betsycrowe@example.com a/Newgate Prison p/1234567 t/criminal b/1920-05-06 i/1000000 ra/H`
 
 ### Listing all persons : `list`
```

Comment:
Perhaps we could change this to listContacts to reflect the updated command word scheme.

Reply:
Ah yes I forgot about this thank you!